ArduPlane/AP_Arming.h

```diff
@@ -29,12 +29,16 @@ class AP_Arming_Plane : public AP_Arming
     void update_soft_armed();
     bool get_delay_arming() const { return delay_arming; };
 
+    // mandatory checks that cannot be bypassed.  This function will only be called if ARMING_CHECK is zero or arming forced
+    bool mandatory_checks(bool display_failure) override;
+
 protected:
     bool ins_checks(bool report) override;
     bool terrain_database_required() const override;
 
     bool quadplane_checks(bool display_failure);
     bool mission_checks(bool report) override;
+    bool rc_received_check(bool display_failure);
```

Comment:
needs comment on what this does

Reply:
added `// Checks rc has been received if it is configured to be used`